### ISSUE TITLE
terraform plan truncated in github actions comments

### ISSUE THREAD

**Issue by nbugden** [terraform-google-modules/terraform-example-foundation#1130]
### TL;DR

The terraform plan generated in GitHub Actions is often truncated when the comment is posted to a PR. This is because the comment is posting the stdout for all environments not just the plan generated by terraform.

### Expected behavior

Log the terraform plan as a separate text file for each environment and then use those files to create 1 comment per environment on the PR.

### Observed behavior

One comment is created on the PR for all environments. This comment includes unnecessary logs related to the init which uses characters that could have been reserved for the plan.

### Terraform Configuration

```hcl
n/a
```

### Terraform Version

```sh
n/a
```

**Comment by nbugden** [comment]
Still open

**Comment by eeaton** [comment]
Fixed in #1129
